package.json

```diff
@@ -201,7 +201,7 @@
     "@typescript-eslint/parser": "^4.9.0",
     "auto-changelog": "^2.2.1",
     "axios-mock-adapter": "^1.19.0",
-    "babel-loader": "^8.2.2",
+    "babel-loader": "^8.2.1",
```

Comment:
snyk wizard recommended a version downgrade?

Reply:
package [seems ok](https://snyk.io/test/npm/babel-loader/8.2.2)

Reply:
snyk works in mysterious ways